app/scripts/lib/validate.js

```diff
@@ -69,8 +69,8 @@ define(function (require, exports, module) {
       }
 
       // codes are fixed length hex strings.
-      return code.length === Constants.CODE_LENGTH &&
-        HEX_STRING.test(code);
+      return (code.length === Constants.CODE_LENGTH || code.length === 64) &&
```

Comment:
Whoops, ignore that, I failed to read your comment about removing `|| code.length === 64`.